app/static/src/app/store/graphSettings/state.ts

```diff
@@ -0,0 +1,3 @@
+export interface GraphSettingsState {
```

Comment:
This needs pulling through the serialisation code, currently the setting is not preserved on save/load